main.c

```diff
@@ -508,13 +509,18 @@ main(int argc, char **argv)
 		"  --border-color <#argb>     Define border color\n"
 		"  --background-color <#argb> Define background color\n"
 		"  --bar-color <#argb>        Define bar color\n"
+		"  -f, --allow-overflow       Allow values over maximum\n"
+		"  --overflow-color <#argb>   Define bar color when overflowed"
```

Comment:
It would be great to have `--overflow-bar-color` as well as `--overflow-border-color` and `--overflow-background-color` if you are up to that task :slightly_smiling_face: Otherwise just rename `--overflow-color` to `--overflow-bar-color` and leave it at that.

Reply:
I thought about this initially but wasn't sure if it was wanted so I just left it on the table. I'll add this, no problem.

Reply:
Done.